modules/kubernetes/grafana-agent/charts/grafana-agent-extras/values.yaml

```diff
@@ -0,0 +1,10 @@
+image: grafana/agent:v0.21.2
+replicas: 2
+environment: dev
+clusterName: aks1
+credentialsSecretName: grafana-agent-credentials
+
+remote:
+  metricsUrl: https://prometheus-prod-01-eu-west-0.grafana.net/api/prom/push
+  logsUrl: https://logs-prod-eu-west-0.grafana.net/loki/api/v1/push
+  tracesUrl: tempo-eu-west-0.grafana.net:443
```

Comment:
NIT: Should we have default values for EU or just set empty strings instead?

Reply:
fixed!